Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -327,6 +331,23 @@ final class MapViewTests: XCTestCase {
         XCTAssertEqual(attributionURLOpener.openAttributionURLStub.invocations.count, 1)
         XCTAssertEqual(attributionURLOpener.openAttributionURLStub.invocations.first?.parameters, url)
     }
+
+    func testEventsFlushingOnDeinit() throws {
+        dependencyProvider.makeEventsManagerStub.returnValueQueue.append(EventsManagerStub())
+
+        autoreleasepool {
```

Comment:
Does this `autorelease` do anything here? Asking because the map view is retained by the `mapView` property and it won't be deallocated when existing the scope of the autoreleasepool closure(in other tests it's a bit different in a way that the map view is created and deallocated in the scope of autoreleasepool)

Reply:
Actually, it is!
As in the other tests, `mapView` get hold by the autoreleasepool during creation and configuration phase. To enforce that strong reference to destroy I'm forcing autorelease pool context.

Reply:
Interesting, I had a different idea about how it works, TIL